iron-doc-property.html

```diff
@@ -93,6 +93,15 @@
         observer: '_collapsedChanged',
       },
 
+      /**
+       * Unique anchor ID for deep-linking.
+       *
+       */
+      id: {
```

Comment:
this might collide with the `id` attribute coming from the prototype of `HTMLElement`. In the past, we had issues with these attributes. e.g. `id`, `name`.